src/apis/GooglePlay.js

```diff
@@ -0,0 +1,18 @@
+const { APIWrapper } = require('../')
+const gPlay = require('google-play-scraper')
+
+module.exports = class GooglePlayStore extends APIWrapper {
+  constructor () {
+    super({
+      name: 'gplaystore'
+    })
+  }
+
+  async searchApp (query) {
+    try {
+      return await gPlay.search({ term: query, num: 10, fullDetail: true })
+    } catch (e) {
+      return new Error('Not found')
+    }
+  }
+}
```

Comment:
Wouldn't only returning the promise straight away be a better idea (without try/catching it)? We could then catch any errors with `.catch` in the command instead

Reply:
Making every command(If another is made in the future) that uses the API having to handle itself?

Unless all the APIs work like you suggested, I don't see why make this move.

Reply:
Well, we'd still have to do something with the Error you're returning. Promises already have rejection, you're just wrapping it in another layer of complexity.

Reply:
Actually, this whole API file is quite redundant, as this library requires no tokens or configuration and could be used directly in the command instead

Reply:
I mean, the API is kinda glitchy, I already told you in the whatsapp